src/components/Footer/index.tsx

```diff
@@ -0,0 +1,105 @@
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import CONFIG from '@/lib/configs';
+import { IconType } from '@/lib/icons';
+import { Flex } from '@/styles/containers';
+import { P } from '@/styles/text';
+import FooterImage from '../../../public/images/footer-image.webp';
+import Icon from '../Icon';
+import * as Styles from './styles';
+
+type SocialLink = {
+  href: string;
+  type: IconType;
+};
+
+const socialLinks: SocialLink[] = [
+  {
+    href: CONFIG.facebook,
+    type: 'facebook',
+  },
+  { href: CONFIG.xTwitter, type: 'x' },
+  {
+    href: CONFIG.linkedin,
+    type: 'linkedin',
+  },
+  { href: CONFIG.instagram, type: 'instagram' },
+];
+
+export default function Footer() {
+  return (
+    <Styles.FooterContainer>
+      <Styles.FooterDiv>
+        <Styles.FooterSection>
+          <Image
+            alt="footer image"
+            src={FooterImage.src}
+            placeholder="blur"
+            blurDataURL={FooterImage.src}
+            quality={100}
+            width={168}
+            height={66}
+          />
+          <hr style={{ width: '160px', color: 'white' }} />
+          <P $color="white">
+            2727 Camino del Rio South, Suite 320 San Diego, CA 92108
+          </P>
+          <P $color="white">©2024 by ABA Immigration Justice Project</P>
+          <Flex
+            $direction="row"
+            $gap="10px"
+            $py="40px"
+            $px="0px"
+            $align="center"
+          >
+            <Styles.FooterLink
+              $color="white"
+              href="https://calblueprint.org/"
+              style={{ fontSize: '0.938rem' }}
```

Comment:
we can remove this inline style since the font size is already included in the definition of `FooterLink`